qubesmanager/log_dialog.py

```diff
@@ -52,8 +51,32 @@ def __init__(self, app, log_path, parent=None):
         self.__init_log_text__()
 
     def __init_log_text__(self):
+        butts_in_row = 3
```

Comment:
:smile: 
```suggestion
        btns_in_row = 3
```

Reply:
lol

Reply:
But you need to update usages of the variable too.

Reply:
Yes sorry, it seems that there are also conflicts with other PR. I will check tonight.